scripts/ci.ps1

```diff
@@ -59,7 +59,7 @@ if (! $NoTest) {
         -TestsPath "$REPO_DIR/out/test/unit-tests" `
         -Workspace "$REPO_DIR/test/unit-tests/test-project-without-cmakelists"
 
-    foreach ($name in @("successful-build"; )) {
+    foreach ($name in @("successful-build"; "without-cmakelist-file"; )) {
```

Comment:
Oh noes. Did I accidentally commit out the tests? Can you make sure that all the extension tests are listed here?